src/lib/services-manager/widgets/logs_button.rb

```diff
@@ -26,9 +26,14 @@ module Y2ServicesManager
   module Widgets
     # Button to show logs of a service
     class LogsButton < Base
-      extend Yast::I18n
 
-      textdomain "services-manager"
```

Comment:
Does it work? I had understood that textdomain needs to be indicated by file, not by class. So, putting it in the base class would not be enough.